.goreleaser.yaml

```diff
@@ -7,6 +7,13 @@ builds:
   goarch:
   - amd64
   - arm64
+  ldflags:
+      - "-s -w"
```

Comment:
White space looks off. This is supposed to be a YAML list right?

Reply:
Fixed, thank you. ✌️